bucket/btop-lhm.json

```diff
@@ -0,0 +1,38 @@
+{
+    "version": "1.0.3",
+    "description": "A monitor of resources, btop++ for windows",
+    "homepage": "https://github.com/aristocratos/btop4win",
+    "license": "Apache-2.0",
+    "notes": [
+        "Has full GPU monitoring, etc. using Libre Hardware Monitor, DLL's included in the package.",
+        "Requires admin rights to run."
+    ],
```

Comment:
```suggestion
    "notes": "This application requires admin rights to run.",
```

Reply:
I think keeping the notes from the repo is better, as they explain more what the differences are between the manifests